dpnp/dpnp_iface_logic.py

```diff
@@ -1196,6 +1197,95 @@ def isfortran(a):
     return a.flags.fnc
 
 
+def isin(
+    element,
+    test_elements,
+    assume_unique=False,  # pylint: disable=unused-argument
+    invert=False,
+):
+    """
+    Calculates ``element in test_elements``, broadcasting over `element` only.
+    Returns a boolean array of the same shape as `element` that is ``True``
+    where an element of `element` is in `test_elements` and ``False``
+    otherwise.
+
+    For full documentation refer to :obj:`numpy.isin`.
+
+    Parameters
+    ----------
+    element : {dpnp.ndarray, usm_ndarray, scalar}
+        Input array.
+    test_elements : {dpnp.ndarray, usm_ndarray, scalar}
+        The values against which to test each value of `element`.
+        This argument is flattened if it is an array.
+    assume_unique : bool, optional
+        Ignored, as no performance benefit is gained by assuming the
+        input arrays are unique. Included for compatibility with NumPy.
+        Default: ``False``.
+    invert : bool, optional
+        If ``True``, the values in the returned array are inverted, as if
+        calculating `element not in test_elements`.
+        ``dpnp.isin(a, b, invert=True)`` is equivalent to (but faster
+        than) ``dpnp.invert(dpnp.isin(a, b))``.
+        Default: ``False``.
```

Comment:
an empty line is missing:
```suggestion
        than) ``dpnp.invert(dpnp.isin(a, b))``.

        Default: ``False``.
```